bluebottle/clients/management/commands/bulk_import.py

```diff
@@ -166,7 +170,11 @@ def _handle_projects(self, data):
         except Project.DoesNotExist:
             project = Project(slug=data['slug'])
 
-        project.owner = Member.objects.get(email=data['user'])
+        try:
+            project.owner = Member.objects.get(email=data['user'])
+        except Member.DoesNotExist:
+            project.owner = Member.objects.all()[0]
```

Comment:
What is the first user normally? Maybe we can make a special user for this purpose.

Reply:
Yeah, it's a bit lame like this, but it also weird to do a look up for 'info@voorjebuurt.nl' here. Maybe use admin@example.com?